### DISCUSSION TITLE
Welcome to Suggestions!

### DISCUSSION THREAD

**Discussion by ArcerionDev** [Prodigy-Hacking/ProdigyMathGameHacking#404] [(C) Suggestions]
Suggest things 🤷🏼

**Comment by ghost** [comment]
Hey I’m on mobile and I can’t use the extension. Do you have a bookmark let for mobile? If you don’t that’s my suggestion.

**Reply by yokisniff** [reply]
I have an idea...download the puffin browser from the app store because you can add extensions

**Reply by PatheticMustan** [reply]
Ehhh? Does puffin even support chrome plugins?

**Reply by ghost** [reply]
> Ehhh? Does puffin even support chrome plugins?

Question is does it even have dev mode for extensions?

**Reply by yokisniff** [reply]
> Ehhh? Does puffin even support chrome plugins?

@PatheticMustan @XxDankGod96xX  it supports what chrome supports and yes it does have dev mode

**Comment by PatheticMustan** [comment]
Unfortunately bookmarklets only are impossible. :(

**Reply by ghost** [reply]
*cries in reality*

**Reply by ghost** [reply]
it might be possible to use script-based hacks but its kinda complicated

**Comment by yokisniff** [comment]
can you make a hack level that makes your level like 99.999 and public to everyone? I saw some people with that

**Reply by ArcerionDev** [reply]
level hacks aren't public to others unless you're using the level hack yourself

**Reply by yokisniff** [reply]
wdym

**Reply by PixelMMO** [reply]
why?

**Reply by PixelMMO** [reply]
the battle health thing also doesnt show to public

**Comment by PixelMMO** [comment]
you should add a hack that completes the dark tower and an in game hack that does arena points, i cant open the tab for the arena points it says there is a fire wall

**Reply by PixelMMO** [reply]
you should also rename all the teleport map locations it makes me confused

**Reply by irhrhd** [reply]
I have a bot that does arena points

**Reply by irhrhd** [reply]
> you should also rename all the teleport map locations it makes me confused

https://discord.gg/QusT2R3pnW Theres the bot

**Reply by PixelMMO** [reply]
i see the arena point bot but not the map bot

**Comment by irhrhd** [comment]
I can add a map bot if you want

**Reply by PixelMMO** [reply]
please do

**Comment by PixelMMO** [comment]
you should add a hack that kills titan

**Reply by PixelMMO** [reply]
and a hack that allows you to say anything you want in the chat box

**Comment by irhrhd** [comment]
Okay ill fix that

**Comment by irhrhd** [comment]
Me?

**Reply by PixelMMO** [reply]
wdym

**Reply by PixelMMO** [reply]
a hack that makes people with the hacks able to say things that you shouldnt be able to say

**Comment by irhrhd** [comment]
I might be able to do that stuff you want

**Reply by PixelMMO** [reply]
ty

**Reply by PixelMMO** [reply]
the arena point thing isnt working, i still have 1000 like i had before

**Reply by PixelMMO** [reply]
if you can make custom nick names can you make a nickname like W12-4RD but instead its a blank space so it only says the first name

**Comment by PixelMMO** [comment]
when i press the morph hack it crashes the game even if i press "fix morph crash" before it

**Reply by PixelMMO** [reply]
new picture POG

**Comment by PixelMMO** [comment]
NEW PICTURE POG

**Reply by PixelMMO** [reply]
why isnt it showing :(

**Reply by PixelMMO** [reply]
can you set >‏‏‎ ‎‏‏‎ ‎‏‏‎ ‎< as a nickname?

the nickname request is between the arrows

**Reply by PixelMMO** [reply]
so that my name can just be <name> with nothing at the end

**Comment by PixelMMO** [comment]
‏‏‎ ‎

**Reply by PixelMMO** [reply]
that should be a nickname

**Comment by NessiePUYOY** [comment]
can you please make it possible to get rare hairstlyles and add it to the cheat menu please and thank you it would help alot :D

**Reply by PatheticMustan** [reply]
ok

**Comment by NessiePUYOY** [comment]
Also it would be cool if we can say other things than just the stuff that people can say in the chat box, it would be cool to say things other people can not say

**Reply by ghost** [reply]
Yeah I want to be able to say whatever in the messages area.

**Reply by PatheticMustan** [reply]
unfortunately not possible, we've tried before, but Prodigy whitelists chat messages

**Reply by ArcerionDev** [reply]
I feel like it did work at first, but then prodigy changed it up...?

**Comment by ghost** [comment]
Also can you make it so that you can type in any name that you want when you are choosing your name so that we do not have to choose one of the ones that are already there.

**Reply by PixelMMO** [reply]
agreed

**Reply by PixelMMO** [reply]
i want my name to just be pixel

**Comment by PixelMMO** [comment]
do you just update the extension that already exists or do i have to reinstall the extension?

**Comment by PixelMMO** [comment]
instant kill doesnt instant kill for me, it just does normal damage

**Reply by PixelMMO** [reply]
@PatheticMustan

**Reply by PatheticMustan** [reply]
Sad

**Comment by PixelMMO** [comment]
when i try to set my level above 100 it works but then it reverts back to lvl 100 

example: when i set my lvl to 1e+100 it works but it reverts back to lvl 100 later

**Comment by PixelMMO** [comment]
add an arena hack to change the amount of arena points you have (please put in menu)

**Reply by PixelMMO** [reply]
please dont make it a new tab again, when i tried to open that tab it said malicious content was in that site

**Comment by PixelMMO** [comment]
you should make a prodigy classroom for all the people with hacks

**Reply by ArcerionDev** [reply]
why

**Reply by PixelMMO** [reply]
im bored of fighting normal players its to ez

**Reply by PixelMMO** [reply]
i wanna fight a haxer

**Comment by PixelMMO** [comment]
***suggestion***
make a hack that auto answers questions

***location***
put the hack under the battle hacks

***how it works***
when you get into the area in a battle where you have to answer a question you open the menu and press a button that closes that area and auto answers the question

**Reply by PixelMMO** [reply]
when i open the menu in the question area i cant press the buttons please fix that

**Comment by PixelMMO** [comment]
add a button in the menu that kills the titan

**Comment by NessiePUYOY** [comment]
It would be amazing if you allowed us to have rare hairstyles i would be so happy if it could be possible <3

**Reply by PixelMMO** [reply]
just buy normal hair styles

**Reply by PatheticMustan** [reply]
No

**Comment by PixelMMO** [comment]
when do the devs update and add things to the cheat menu?

**Reply by PatheticMustan** [reply]
When we feel like it

**Reply by ArcerionDev** [reply]
when we have nothing else to do

**Reply by PixelMMO** [reply]
やれやれだぜ

**Comment by PixelMMO** [comment]
やれやれだぜ

**Comment by PixelMMO** [comment]
can you add Titans to the pet list?

if yes please add them to the ***give all pets hack*** and the ***add pet hack*** list

**Reply by PixelMMO** [reply]
it would be so cool to use them in a battle

**Reply by PatheticMustan** [reply]
no

**Reply by PixelMMO** [reply]
aww ok :(

**Comment by PixelMMO** [comment]
are you changing the name of the cheat menu?

**Comment by PixelMMO** [comment]
you should make a hack that automatically completes tower town (please put in menu)

**Reply by PixelMMO** [reply]
@PatheticMustan

**Reply by PatheticMustan** [reply]
Quest completion

**Reply by PixelMMO** [reply]
uhhh

**Reply by PixelMMO** [reply]
im using wills cheat menu is that the right cheat menu?

**Comment by PixelMMO** [comment]
you should just rename plump snup to **snup on marijuana**

**image**
![image](https://user-images.githubusercontent.com/79719521/110142279-ff390400-7da3-11eb-98a8-8d28fafb937e.png)

**Reply by PixelMMO** [reply]
seems like its on a depressant XD

**Reply by ghost** [reply]
> seems like its on a depressant XD

a sedative

**Reply by PixelMMO** [reply]
YES XD

**Reply by PixelMMO** [reply]
PERFECT

**Reply by PixelMMO** [reply]
UPVOTE IT

**Reply by ghost** [reply]
the staff will be super pissed about this kek

**Reply by PixelMMO** [reply]
ikr

**Reply by PixelMMO** [reply]
@ArcerionDev what do you think?

**Reply by PixelMMO** [reply]
brb

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/61570792/110143001-9c3b7300-7d8b-11eb-8f28-b648d529b818.png)
OK IM NO LONGER TROLLING IN DISCUSSIONS

**Reply by PixelMMO** [reply]
back
![image](https://user-images.githubusercontent.com/79719521/110143251-09a7cd80-7da5-11eb-8536-4c59ecb65aef.png)

**Reply by PixelMMO** [reply]
prodigy meth game XD

**Comment by PixelMMO** [comment]


**Reply by PixelMMO** [reply]
please add this

**Reply by PixelMMO** [reply]
:(

**Comment by PixelMMO** [comment]
‎
